style.css

```diff
@@ -94,3 +112,180 @@ section .social-media-links {
 section .icon-right {
   margin-right: 20px;
 }
+
+.works {
+  background-image: none;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: minmax(150px, auto);
+}
+
+.works section {
+  margin-bottom: 70px;
+  border-radius: 20px;
+  border: 0.2rem solid #e5e5e5;
+  border-color: #e5e5e531;
+  padding: 16px 30px 25px 16px;
+  background-color: white;
+  background-image: none;
+  margin-left: 3px;
+}
+
+.works section .section-title {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  margin-top: 10px;
+  width: 200px;
+}
+
+.works section img {
+  width: auto;
+  cursor: pointer;
+}
+
+.works section .section-about {
+  display: flex;
+  height: 32px;
+  width: 300px;
+  margin-top: 4px;
+  align-items: center;
+  padding: 8px auto;
+  gap: 12px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.works section .section-about .size {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.works section .section-about .back-end {
+  color: #7a869a;
+  height: 16px;
+}
+
+.works section .section-about .date {
+  color: #7a869a;
+}
+
+.works section .section-about a {
+  color: #344563;
+  font-weight: 600;
+}
+
+.works section p {
+  color: #344563;
+  font-weight: 400;
+}
+
+.works section ul {
+  display: flex;
+  list-style: none;
+  align-items: flex-start;
+  margin-left: -40px;
+  gap: 8px;
+  height: 24px;
+}
+
+.works section ul li {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+}
+
+.about-myself button {
+  width: 120px;
+  height: 44px;
+  color: #6070ff;
+  background-color: white;
+  border-color: #6070ff;
+  border-radius: 10px;
+  border: 0.1rem solid;
+}
+
+.works section button {
+  width: 95px;
+  height: 44px;
+  color: #6070ff;
+  background-color: white;
+  border-color: #6070ff;
+  border-radius: 10px;
+  border: 0.1rem solid;
+}
```

Comment:
- [ ] For all `see project` buttons and `get my resume` button please add hover and active(pressed) interactions like the given design.

![image](https://user-images.githubusercontent.com/68796232/170547728-dc7db75d-fbd2-41c5-ac7d-522bedfde88d.png)